source/grid/grid_reordering.cc

```diff
@@ -733,7 +733,7 @@ GridReordering<2>::invert_all_cells_of_negative_grid(const std::vector<Point<2>
       // in lexicographic ordering
       for (unsigned int i=0; i<GeometryInfo<2>::vertices_per_cell; ++i)
         vertices_lex[GeometryInfo<2>::ucd_to_deal[i]]=cells[cell_no].vertices[i];
-      if (GridTools::cell_measure<2>(all_vertices, vertices_lex) < 0)
+      if (GridTools::cell_measure(all_vertices, vertices_lex) < 0)
```

Comment:
It always calls the right one here since we have `dim == spacedim` and `...` has lowest overload precedence. This part does not need to be changed: I will revert it.